Source/Resources/Content/MSGraph/DeviceManagement/DeviceEnrollmentConfigurations/deviceEnrollmentLimitConfiguration - All users and all devices.json

```diff
@@ -7,7 +7,7 @@
       }
     }
   ],
-  "description": "This is the default Device Limit Restriction applied with the lowest priority to all users regardless of group membership.",
+  "description": "What is it: The default Device Limit Restriction applied with the lowest priority to all users regardless of group membership. \n Learn More: https://docs.microsoft.com/en-us/mem/intune/enrollment/enrollment-restrictions-set#create-a-device-limit-restriction",
```

Comment:
**What does this do?**
Limits the default number of devices an individual can enroll to 5. This limit can be overridden by other configurations with a higher priority.

**Why should you use this?**
If you want to ensure that no more than 5 devices are enrolled by a single user so as to prevent the enrollment of unwanted devices. 

**What is the end user impact?**
Users will be limited to enrolling 5 devices.

**Learn more**
[Create a device limit restriction](https://docs.microsoft.com/en-us/mem/intune/enrollment/enrollment-restrictions-set#create-a-device-limit-restriction)